pkg/authentication/scram/scram.go

```diff
@@ -302,7 +302,7 @@ func convertMongoDBResourceUsersToAutomationConfigUsers(secretGetUpdateCreateDel
 	for _, u := range mdb.GetScramUsers() {
 		acUser, err := convertMongoDBUserToAutomationConfigUser(secretGetUpdateCreateDeleter, mdb.NamespacedName(), u)
 		if err != nil {
-			return nil, err
+			return nil, errors.Errorf("failed to convert user %s: %s", u.Username, err)
```

Comment:
[nit] maybe add "to automation config user" to make it clearer?

Reply:
yep good idea!